cmd/cueckoo/cmd/cltrigger.go

```diff
@@ -193,12 +192,20 @@ func (c *cltrigger) triggerBuilds(revs []revision) error {
 }
 
 func (c *cltrigger) triggerBuild(rev revision) error {
-	in, err := c.cfg.gerritClient.GetChange(context.Background(), rev.changeID, gerrit.QueryChangesOpt{
+	query := fmt.Sprintf("change:%s is:open", rev.changeID)
```

Comment:
I'd add a comment here to explain why this works with cherry picks. It took me a couple of minutes of thinking to understand that this was the relevant part of the change and why it works.

Reply:
isn't the commit message enough? perhaps a valid point that I should briefly document this here too, but I did take the time to carefully explain why it works :)

Reply:
GitHub being GitHub, I didn't even review the commit message (yet).

That explanation makes sense... but I think it's also worth mentioning here too, albeit briefly. Or copy the commit message.

Out of interest, what happens if we have two backports open?

Reply:
> Out of interest, what happens if we have two backports open?

Then we'll error with the "found multiple" message I added. I assumed that we won't really run into this scenario for now. If we do need it to work in the future, we can lift that restriction and loop over the resulting changes instead.

> GitHub being GitHub, I didn't even review the commit message (yet).

I add the "see commit message" at the top to try to remind people, but it's too easy to miss with their UI still :) I've added a short explanation to the code.

Reply:
Thanks for pointing out the commit message, which is indeed very detailed.